src/index.js

```diff
@@ -40,7 +32,7 @@ function renderTodos() {
    <i class= "fa  ${
   todo.checked ? 'solid fa-check' : 'regular fa-square'
 }" data-action="check"></i>
-   <p class= "text "${todo.checked ? red : ''}  data-action="check">${
+   <p class= "text "${todo.checked ? 'checked' : ''}  data-action="check">${
   todo.value
 }</p>
     <input type="text" class="edit-input hidden" value=${todo.value}/>
```

Comment:
Good job! It looks great, but when I try to edit a task it adds a slash `/` to the end of the value. You can fix it by putting a space between the todo.value and the closing slash like this `value=${todo.value} />`.

![image](https://user-images.githubusercontent.com/34779928/195919975-ea29d21c-96dd-4e36-ae0e-7c0ac090f590.png)

Reply:
cool thanks for the tip fixed